src/config/artists.rs

```diff
@@ -1,17 +1,30 @@
 use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Default, Clone)]
+#[derive(Debug, Clone)]
 pub struct Artists {
     pub album_display_mode: AlbumDisplayMode,
     pub album_sort_by: AlbumSortMode,
+    pub album_date_tags: Vec<String>,
 }
 
-#[derive(Debug, Default, Clone, Serialize, Deserialize, PartialEq, Eq)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub struct ArtistsFile {
     #[serde(default)]
     pub album_display_mode: AlbumDisplayMode,
     #[serde(default)]
     pub album_sort_by: AlbumSortMode,
+    #[serde(default = "default_album_date_tags")]
+    pub album_date_tags: Vec<String>,
```

Comment:
I think this is better represented as an enum
```suggestion
    pub album_date_tags: Vec<AlbumDateTag>,
```

and then impl the enum and adjust the default etc accordingly: 

```rust
#[derive(Debug, Default, Clone, Copy, Serialize, Deserialize, PartialEq, Eq, Hash, strum::IntoStaticStr)]
#[strum(serialize_all = "lowercase")]
pub enum AlbumDateTag {
    #[default]
    Date,
    OriginalDate,
}
```

This will ensure that no other value can get in there via config file or other means, the `strum::IntoStaticStr` coupled with the `lowercase` attribute will enable you to get the enum value as a `&'static str` - to be used when getting values out of the map.